apps/web/components/FeedbackFormCard.tsx

```diff
@@ -43,20 +33,28 @@ export const FeedbackFormCard: React.FC<Props> = ({ id, address }) => {
   }, [wallet]);
 
   const getTitle = async () => {
-    const title = await feedbackForm.title();
-    setTitle(title);
+    if (isConnected) {
+      const contract = getFormContractClientInstance();
```

Comment:
When you call `getFormContractClientInstance` you're fetching the Factory Contract, and here you should be fething the Feedback Form contract instead.

The method `getFormContractClientInstance` can receive a parameter that can be the address and use `FEEDBACK_FACTORY_CONTRACT_ADDRESS` as default.